### ISSUE TITLE
Error: Cannot find module './helpers'

### ISSUE THREAD

**Issue by ex1st** [mgcrea/node-xlsx#109]
Node version: v10.15.1
node-xlsx version: 0.14.0

```
Error: Cannot find module './helpers'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/home/user/folder/node_modules/node-xlsx/lib/index.js:20:16)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

I can't find this file in module folder also, only `helpers.js.map`

**Comment by superlbr** [comment]
fallback to 0.12.x is fine

**Comment by mgcrea** [comment]
Sorry guys, botched the last release, should be fixed with `v0.14.1`.

Thanks.
